src/include/btree.i

```diff
@@ -1707,3 +1707,63 @@ __wt_ref_state_yield_sleep(uint64_t *yield_count, uint64_t *sleep_count)
 	(*sleep_count) = WT_MIN((*sleep_count) + 100, WT_THOUSAND);
 	__wt_sleep(0, (*sleep_count));
 }
+
+/*
+ * __wt_stat_read_op_histogram --
+ * 	Add an operations execution time to the read operation histogram.
+ */
+static inline void
+__wt_stat_read_op_histogram(WT_SESSION_IMPL *session, uint64_t usecs)
+{
+	/*
+	 * Ignore any operation that takes less than 100us to execute. This
+	 * floor value keeps us from having an excessively large smallest
+	 * execution time bucket. The 100us floor along with the other
+	 * histogram values were chosen here based on testing but performance
+	 * will vary considerably.
+	 */
+	if (usecs  < 100)
+		return;
+
+	if (usecs < 250) {
```

Comment:
KNF: All of the curly braces here and in the other functions should be removed. These are single statement conditionals.